rbac/internal/views.py

```diff
@@ -898,3 +898,56 @@ def username_lower(request):
             )
         Principal.objects.bulk_update(principals, ["username"])
         return HttpResponse(f"Updated {len(principals)} usernames", status=200)
+
+
+def principal_removal(request):
+    """Get/Delete not active principals.
+
+    GET or DELETE /_private/api/utils/principal/?usernames=a,b,c
+    """
+    logger.info(f"Principal edit or removal: {request.method} {request.user.username}")
+    if request.method not in ["DELETE", "GET"]:
+        return HttpResponse('Invalid method, only "DELETE" or "GET" is allowed.', status=405)
+
+    if not request.GET.get("usernames"):
+        return HttpResponse("Please provided a list of usernames with comma separated.", status=400)
+    usernames = request.GET.get("usernames").split(",")
+    resp = PROXY.request_filtered_principals(usernames, org_id=None, options={"return_id": True})
+
+    if isinstance(resp, dict) and "errors" in resp:
+        return HttpResponse(resp.get("errors"), status=400)
+
+    active_users = {(principal_data["username"], principal_data["org_id"]) for principal_data in resp["data"]}
```

Comment:
Are 'active' users those who exist in the IT service and can be retrieved by BOP? These users could be either enabled or disabled, correct?

Reply:
Active users are enabled users. BOP only return enabled usres